frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.tsx

```diff
@@ -91,7 +90,9 @@ export const CardsContainer = styled(FullWidthContainer)`
 `;
 
 function getParametersWidgetBgColor(isNightMode: boolean) {
-  return isNightMode ? color("bg-black") : color("bg-light");
+  return isNightMode
+    ? "color-mix(in srgb, var(--mb-color-bg-black), var(--mb-color-embed-bg-color-override, var(--mb-color-bg-black))"
```

Comment:
where is `--mb-color-embed-bg-color-override` set? I couldn't find it being defined anywhere in our codebase.